tests/SingleImmutableValueObjectTest.php

```diff
@@ -4,26 +4,19 @@
 
 namespace NorseBlue\ValueObjects\Tests;
 
-use Exception;
 use NorseBlue\HandyProperties\Exceptions\PropertyNotMutableException;
 use NorseBlue\ValueObjects\Tests\Helpers\SingleImmutableVO;
 
 class SingleImmutableValueObjectTest extends TestCase
 {
     /** @test */
-    public function immutable_single_value_property_object_cannot_change_value()
+    public function immutable_single_value_property_object_cannot_change_value(): void
     {
         $subject = new SingleImmutableVO();
 
-        try {
-            $subject->value = 9;
-        } catch (Exception $e) {
-            $this->assertInstanceOf(PropertyNotMutableException::class, $e);
-            $this->assertEquals('value', $e->getProperty());
+        $this->expectException(PropertyNotMutableException::class);
```

Comment:
Although PHPUnit has the `expectException` and `expectExceptionMessage` methods I prefer the explicit way as it gives you more flexibility, in the long run, to assert against the status of your application when having an exception. Maybe, in this case, it seems excessive as we are not asserting anything else of relevance, but still, I prefer to have this already set up in case it's needed.

Please refer to this tweet: https://twitter.com/adamwathan/status/823598854536830976